go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -117,7 +117,10 @@ func newNodeTracker(nodeInformer coreinformers.NodeInformer) *nodeTracker {
 			nt.removeNodeWithServiceReSync(node.Name)
 		},
 	}))
-
+	if err != nil {
+		klog.Fatalf("Could not add Event Handler during nodeTracker create %v", err)
```

Comment:
this needs to return the error. Services controller calls newNodeTracker and you added error to NewController so its able to be propagated.